pkg/scheduler/api/node_info.go

```diff
@@ -122,6 +122,10 @@ func (ni *NodeInfo) Ready() bool {
 }
 
 func (ni *NodeInfo) setRevocableZone(node *v1.Node) {
+	if node == nil {
+		return
```

Comment:
let's log a message here, e.g. warning. BTW, will scheduler re-update revocable-zone after that?

Reply:
I checked it ， and I found it drop the setRevocableZone in function SetNode and I will add it。
And I add some log in function setNodeGPUInfo, setNodeState, setRevocableZone